projects/devices/robotiq/protos/Robotiq3fGripper.proto

```diff
@@ -498,16 +503,17 @@ PROTO Robotiq3fGripper [
                         device [
                           RotationalMotor {
                             name "finger_middle_joint_3"
-                            maxVelocity 100
-                            maxPosition 3.1416
-                            maxTorque 100
+                            maxVelocity 100.0
+                            minPosition -1.1694
```

Comment:
Doesn't match too:
https://github.com/ros-industrial/robotiq/blob/66961ec6b6c9c493f13410d16ce32cea1648babf/robotiq_3f_gripper_visualization/cfg/robotiq-3f-gripper_finger_articulated_macro.xacro#L127

Reply:
Same reasoning as above, lower - upper and rotating endpoint by lower amount